.github/renovate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:base"],
+  "includePaths": ["packages/create-plugin/templates/**"],
+  "separateMajorMinor": false,
+  "labels": ["dependencies"],
+  "reviewers": ["team:grafana/plugins-platform-frontend"],
+  "enabledManagers": ["regex"],
+  "customManagers": [
+    {
+      "customType": "regex",
+      "fileMatch": ["_package.json"],
+      "matchStrings": ["\"(?<depName>@grafana/.*)\"[^\"]*:\\s*\"\\^(?<currentValue>.*)\""],
+      "datasourceTemplate": "npm"
+    },
+    {
+      "customType": "regex",
+      "fileMatch": ["(^|/)(?:docker-)?compose[^/]*\\.ya?ml$"],
+      "matchStrings": ["grafana_version:\\s\\$\\{GRAFANA_VERSION:-(?<currentValue>.*)\\}"],
+      "depNameTemplate": "grafana/grafana-enterprise",
+      "datasourceTemplate": "docker"
+    }
+  ],
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["@grafana/*", "grafana/grafana-enterprise"],
+      "matchUpdateTypes": ["minor", "patch", "major"],
```

Comment:
If we're using tilde ranges in package.json `^10.3.0` do we need update PRs in this repo for patch releases of Grafana? 🤔

Reply:
Let's leave this as other @grafana packages benefit from patch bumps and it'll only complicate the renovate config.